### ISSUE TITLE
Importer V2: delete old Import Templates?

### ISSUE THREAD

**Issue by DavidHepper** [BiologicalRecordsCentre/iRecord#1746]
Import Templates can be overwritten by saving a new definition with the name of an existing one but I cannot see how to delete one.

**Comment by sacrevert** [comment]
I agree that it would be good to have some sort of template management system. One thing that we discussed recently within the BBS database working group was the possibility of being able to import templates, so, e.g. for a specific survey a group could provide the template for users to import (this is because we are promoting a specific recording tool that produces a specific export format).

**Comment by sacrevert** [comment]
Just noting that we have had another request for this functionality via email
